Lib/asyncio/subprocess.py

```diff
@@ -137,8 +137,8 @@ def kill(self):
         debug = self._loop.get_debug()
         self.stdin.write(input)
         if debug:
-            logger.debug('%r communicate: feed stdin (%s bytes)',
-                        self, len(input))
+            logger.debug(
+                '%r communicate: feed stdin (%s bytes)', self, len(input))
```

Comment:
I think this can be made into f-string too?

Reply:
The same